### PR TITLE
Add the ability to set boolean block settings values directly

### PR DESCRIPTION
**Pull request by Kroppeb**
Allows for stuff like `QuiltBlockSettings.copyOf(Blocks.VINE).ticksRandomly(false)`

### CODE REVIEW

File: library/block/block_extensions/src/main/java/org/quiltmc/qsl/block/extensions/api/QuiltBlockSettings.java [hunk 1]
```diff
@@ -265,6 +265,31 @@ public QuiltBlockSettings collidable(boolean collidable) {
...
+		((AbstractBlockSettingsAccessor) this).setDynamicBounds(dynamicBounds);
+		return this;
+	}
+
+	public QuiltBlockSettings requiresTool(boolean requiresTool) {
+		((AbstractBlockSettingsAccessor) this).setToolRequired(requiresTool);
+		return this;
+	}
+
+	public QuiltBlockSettings isAir(boolean isAir) {
```
Comment (LambdAurora) [inline library/block/block_extensions/src/main/java/org/quiltmc/qsl/block/extensions/api/QuiltBlockSettings.java]: I really don't like using a getter-like name for this.
Maybe `airLike(boolean isAir)`?

Comment (Kroppeb) [reply library/block/block_extensions/src/main/java/org/quiltmc/qsl/block/extensions/api/QuiltBlockSettings.java]: Ah, I thought the normal setter was also `settings.isAir()`, but apparently it's `settings.air()`

**Review by Leo40Git — APPROVED** [review]
LGTM!

**Review by OroArmor — APPROVED** [review]

File: library/block/block_extensions/src/main/java/org/quiltmc/qsl/block/extensions/api/QuiltBlockSettings.java (hunk 1 above)
Comment (LambdAurora) [reply library/block/block_extensions/src/main/java/org/quiltmc/qsl/block/extensions/api/QuiltBlockSettings.java]:
```suggestion
	public QuiltBlockSettings air(boolean isAir) {
```

Just change that and I'll be good with FCP-ing this PR.

**Review by LambdAurora — APPROVED** [review]

**Review by EnnuiL — CHANGES_REQUESTED** [review]
A comment on line 51 at QuiltBlockSettings: We could get rid of thisAccessor, right? Also, renaming `otherAccessor` would be interesting

**Review by 0xJoeMama — APPROVED** [review]

**Comment by Kroppeb** [comment]
@EnnuiL 
I got rid of the variable, I don't see why the other one needs to be renamed, but if you want a different name, feel free to suggest one.

**Comment by EnnuiL** [comment]
> @EnnuiL I got rid of the variable, I don't see why the other one needs to be renamed, but if you want a different name, feel free to suggest one.

Yeah, renaming it was only something to consider, however, yeah, maybe the current one is good enough

**Review by EnnuiL — APPROVED** [review]
Let's go!

**Pull request merged**